src/monodroid/jni/dylib-mono.h

```diff
@@ -233,10 +207,10 @@ typedef char*           (*monodroid_mono_method_full_name_fptr) (MonoMethod *met
 typedef MonoClass*      (*monodroid_mono_object_get_class_fptr) (MonoObject *obj);
 typedef MonoObject*     (*monodroid_mono_object_new_fptr) (MonoDomain *domain, MonoClass *klass);
 typedef void*           (*monodroid_mono_object_unbox_fptr) (MonoObject *obj);
-typedef void            (*monodroid_mono_profiler_install_fptr) (void *profiler, void *callback);
-typedef void            (*monodroid_mono_profiler_install_jit_end_fptr) (MonoProfileJitResult end);
-typedef void            (*monodroid_mono_profiler_install_thread_fptr) (void *start_ftn, void *end_ftn);
-typedef void            (*monodroid_mono_profiler_set_events_fptr) (MonoProfileFlags events);
+typedef MonoProfilerHandle (*monodroid_mono_profiler_install_fptr (MonoProfiler *prof);
```

Comment:
This line is missing a `)`; it should be:

    typedef MonoProfilerHandle (*monodroid_mono_profiler_install_fptr) (MonoProfiler *prof);